src/analytics/internal/sendToServer/index.js

```diff
@@ -0,0 +1,9 @@
+import SendToServer from './SendToServer'
+
+const sendToServer = new SendToServer({
+    url:
+        'https://a8495szyaa.execute-api.eu-central-1.amazonaws.com/' +
```

Comment:
This is the same API address as the token generate stuff. I said in the other PR to move that to a constant somewhere, so probably this constant should be re-used in both places. Maybe putting it in the `src/analaytics/internal/constants` module would be best?